app/helpers/api/ordapi/getOrders.test.js

```diff
@@ -31,22 +41,22 @@ describe('getOrders', () => {
     });
   });
 
-  it('should return the expected result', async () => {
-    getData.mockResolvedValueOnce([order1, order2, order3, order4]);
+  it('should return the orders grouped by status and sorted by descending dateCreated or dateCompleted', async () => {
+    getData.mockResolvedValueOnce([order1, order2, order3, order4, order5]);
 
     const { completedOrders, incompletedOrders } = await getOrders({ orgId, accessToken });
-    expect(completedOrders).toEqual([order1]);
-    expect(incompletedOrders).toEqual([order2, order3, order4]);
+    expect(completedOrders).toEqual([order5, order1]);
+    expect(incompletedOrders).toEqual([order3, order4, order2]);
   });
 });
 
 describe('groupOrdersByStatus', () => {
   it('should group complete and incomplete orders', () => {
-    const orders = [order1, order2, order3, order4];
+    const orders = [order1, order2, order3, order4, order5];
 
     const { completedOrders, incompletedOrders } = groupOrdersByStatus(orders);
 
-    expect(completedOrders).toEqual([order1]);
+    expect(completedOrders).toEqual([order1, order5]);
```

Comment:
will this test if it's in the right order or just if they're in the array?

Reply:
this one just tests that it groups the orders correctly. The one above checks that the sort is correct

Reply:
ahh right ok!